Source/sound.cpp

```diff
@@ -8,17 +8,17 @@ LPDIRECTSOUND sglpDS;
 char gbSndInited;
 int sglMusicVolume;
 int sglSoundVolume;
-HMODULE hDsound_dll; // idb
+HMODULE hDsound_dll;
 HANDLE sgpMusicTrack;
 LPDIRECTSOUNDBUFFER sglpDSB;
 
 const int sound_inf = 0x7F800000; // weak
 
 /* data */
 
-UCHAR gbMusicOn = 1;  // weak
-UCHAR gbSoundOn = 1;  // weak
-char gbDupSounds = 1; // weak
+BYTE gbMusicOn = TRUE;
+BYTE gbSoundOn = TRUE;
+BYTE gbDupSounds = TRUE;

```

Comment:
Hmm, was it a typical practice to use TRUE/FALSE for both BYTE/BOOL back in the day?

Reply:
It was suggested to me to do it this way, my original though was that they had some bool=true/false (8bit) here even though it would be inconsistent with the parts of the code that uses BYTE=TRUE/FALSE(32bit).

Reply:
(I strongly favor using some sort of Boolean value as it makes it obvious that we'r not talking about some variable volume or other)